src/partitions.rs

```diff
@@ -0,0 +1,256 @@
+use crate::Error;
```

Comment:
can you add some unit tests for `partitions`?

Reply:
I moved the `Partition` tests from `statics.rs`